Casks/o/openchrom.rb

```diff
@@ -0,0 +1,30 @@
+cask "openchrom" do
+  arch arm: "aarch64", intel: "x86_64"
+
+  version "1.5.0"
+  sha256 arm:   "7fe8d108f47eda1d49b76135be4d2d5a48fe316ce0f945a2fa6df4ccccd7f740",
+         intel: "a09f7f1b2cbf69696149cd5a2db76e445780de1b5f95d28d7b9f71c07cfe4f97"
+
+  url "https://products.lablicate.com/openchrom/#{version}/OpenChrom_#{version}_#{arch}.dmg",
+      verified: "products.lablicate.com/openchrom/"
+  name "OpenChrom"
+  desc "Analytical chemistry data analysis"
+  homepage "https://www.openchrom.net/"
+
+  livecheck do
+    url "https://marketplace.lablicate.com/api/downloads/hash-size"
+    strategy :json do |json|
+      json["data"]["items"]
+        .select { |item| item["name"].include?("openchrom-lablicate_macosx") }
+        .map { |item| item["name"][/(\d+(?:\.\d+)+)/, 1] }
+        .max_by { |version| Gem::Version.new(version) }
+    end
+  end
```

Comment:
I think the livecheck can be simplified down to this, would love @samford's input here!

```suggestion
  livecheck do
    url "https://marketplace.lablicate.com/api/downloads/hash-size"
    regex(/OpenChrom[._-]?v?(\d+(?:\.\d+)+)[._-]#{arch}\.dmg/i)
    strategy :json do |json|
      item = json.dig("data", "items")&.filter_map do |item|
        item["name"]&.[](regex, 1)
      end
    end
  end
```

Reply:
That's how I would approach it 👍

I made one small edit to the regex in the suggestion (making the `[._-]` delimiter after the software name required, as I didn't see any instances in the current file where it's optional) but it looks good to me.

Reply:
I also made a small change to include `regex` in the `strategy` block parameters.